src/components/chatbox/chatbox.tsx

```diff
@@ -377,25 +377,27 @@ const ChatBox = () => {
 							}}
 							id="task"
 							rows={userInput.split('\n').length}
-							placeholder="Send a new message"
+							placeholder="Ask a question"
```

Comment:
Placeholder change addresses [✓ consider adjusting the default language in the input to... "ask a question?" to keep it consistent with the CTAs driving to the experience.](https://app.asana.com/0/0/1205523765248064/f)